pkg/controller/utils/rawext.go

```diff
@@ -1,19 +1,3 @@
-/*
-Copyright 2019 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
```

Comment:
@csrwng was this original code?

Reply:
It is based on the same idea as https://github.com/openshift/hive/blob/master/vendor/k8s.io/cli-runtime/pkg/genericclioptions/printers/typesetter.go but not the same code